packages/cli/src/lib/live-server/index.js

```diff
@@ -281,7 +281,13 @@ LiveServer.start = function(options) {
       setTimeout(function() {
         server.listen(0, host);
       }, 1000);
-    } else {
+    } else if (e.code === 'EADDRNOTAVAIL') {
+      console.log('address(host) %s is not available. Trying another address'.yellow, host);
+      setTimeout(function() {
+        server.listen(port, '0.0.0.0');
```

Comment:
I'll prefer to make 127.0.0.1 as the default address if the host is invalid. Because 0.0.0.0 has some special meaning and might accidentally expose it to the rest of the interner (see [link](https://superuser.com/questions/949428/whats-the-difference-between-127-0-0-1-and-0-0-0-0)).